### ISSUE TITLE
[Channel Configuration] Channel Page is empty on Azure Portal

### ISSUE THREAD

**Issue by phchabrol** [Microsoft/BotBuilder#4407]
## Bot Info
* SDK Platform: .Node.js
* SDK Version:  SDK version  3
* Active Channels: Web and FB messenger 
* Deployment Environment:  Azure Bot Service and local development with Emulator 

## Issue Description
The "channel" page in the Azure Portal is blank. I can't see my channels or update the tokens to change them. 

## Code Example
N/a
## Reproduction Steps
1. I go on my resources page on Azure Protal
2. Click on my bot resource
3. Click on "Channels"

## Expected Behavior
I expect to see the channel screen when I click on channels.

## Actual Results
 It is now empty screen like it's loading forever
<img width="1275" alt="capture d ecran 2018-04-02 a 09 35 13" src="https://user-images.githubusercontent.com/16733885/38187719-30bb3394-3659-11e8-9ea6-00e924779f9d.png"

**Comment by JasonSowers** [comment]
Have you tried logging out and back in, incognito window, clearing cookies, or hard reload?  If you have not please try those and let us know if the problem persists.

**Comment by JasonSowers** [comment]
Well, that is unfortunate.  I **highly** recommend contacting Azure support about this.  I will reach out to our team and see if there is anything on our end that may be causing this.

**Comment by JasonSowers** [comment]
@phchabrol are you able to provide your browser's console output and network trace for us? If so please send them to the email listed in my profile.

**Comment by phchabrol** [comment]
Hi Jason, just dit it! thanks for your help!

**Comment by mgbennet** [comment]
@phchabrol Is this a new issue for this bot? Were you able to see the channel config page in the past, or has it never worked for this particular bot?

Also, can you visit the settings page, and if so is the information accurate?

**Comment by phchabrol** [comment]
hi @mgbennet, yes all seems ok on the "settings" tab.  IT worked very well until yesterday when I tried to connect.

**Comment by mgbennet** [comment]
@phchabrol I'm still looking into why you can't access the channel page, but you should be able to edit the channels by using the devportal. Try going to https://dev.botframework.com/bots/settings?id=YOUR_BOTS_ID, and you should be able to configure your channels. That should unblock you. In the mean time I'll keep digging with regards to Azure.

**Comment by carlos-prado-vsys** [comment]
@phchabrol 

I was facing the very same problem. But I believe I found a way to work-arround....
I was using portuguese as my language at Portal Settings. When I changed do English, the problem disapeared!

Hope it helps.
Carlos Prado.

**Comment by mgbennet** [comment]
Current status of this bug is that certain languages will cause the channels page to throw a reference error. So far the only languages I've found that cause this are French, Polish, and both types of Portuguese. I will investigate further into why only these languages and update further.

**Comment by mgbennet** [comment]
Current status of this bug is that certain languages will cause the channels page to throw a reference error. So far the only languages I've found that cause this are French, Polish, and both types of Portuguese. I've figured out what's causing this and will have a fix soon, though I'm not sure when it will be deployed.

**Comment by mgbennet** [comment]
A fix has been deployed, this issue should be resolved. Please let us know if any other languages have this sort of issue.

**Comment by alexeymkh** [comment]
@mgbennet Hi! I have same issue, but language in my Azure account is English. Also, the same problem is with Test in Web Chat section and Analytics section.
